apps/kitty-krew/Dockerfile

```
@@ -2,35 +2,67 @@ FROM oven/bun:latest AS builder
+RUN echo "Starting build process - builder stage"
+RUN echo "Working directory: $(pwd)"
+
 # Copy package.json and bun.lockb (if available)
+RUN echo "Copying package files"
 COPY package.json ./
 COPY bun.lockb* ./
+RUN echo "Package files copied - contents:"
+RUN ls -la
 
 # Install dependencies
+RUN echo "Installing dependencies with bun"
```

Hadolint (2.12.0)

[info] Multiple consecutive `RUN` instructions. Consider consolidation.

(DL3059)